app/tests/test_token_repository.py

```diff
@@ -0,0 +1,67 @@
+from unittest import TestCase
+from unittest.mock import patch
+
+import boto3
+from docker import from_env, DockerClient
+from docker.models.containers import Container
+
+
+class TestTokenRepository(TestCase):
+    dynamodb_container: Container = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.docker_client = from_env()
+        cls.dynamodb_container_name = 'dynamo-test'
+        cls.dynamodb_container = _run_dynamodb_container(cls.docker_client, cls.dynamodb_container_name)
+        try:
+            dynamodb = boto3.resource('dynamodb', endpoint_url='http://localhost:8000')
+            table = dynamodb.create_table(
+                TableName='hallebarde-dev-table',
+                KeySchema=[
+                    {
+                        'AttributeName': 'id',
+                        'KeyType': 'HASH'
+                    }
+                ],
+                AttributeDefinitions=[
+                    {
+                        'AttributeName': 'id',
+                        'AttributeType': 'S'
+                    }
+                ],
+                BillingMode='PAY_PER_REQUEST',
+            )
+
+            table.meta.client.get_waiter('table_exists').wait(TableName='hallebarde-dev-table')
+        except Exception as e:
```

Comment:
Is there a more specific exception than Exception that we could catch here ?

Reply:
Yes (indeed, many but I do not have the list), but I do not find it useful to be specific. The purpose of this try/except is to make sure that the dynamodb container is removed even if the setup of the table fails. So the tearDown should be triggered in the event of any kind of exception happening while doing the setup of the table.